src/main/webapp/creatives/maps/simid_map_creative.js

```diff
@@ -0,0 +1,55 @@
+
+import BaseSimidCreative from '../base_simid_creative.js';
+
+/**
+ * A sample SIMID ad that shows a map of nearby locations
+ */
+
+const AdParamKeys = {
+  BUTTON_LABEL: 'buttonLabel',
+  SEARCH_QUERY: 'searchQuery',
+  MARKER: 'marker',
+};
+
+const FIND_NEAREST_TEMPLATE_TEXT = "Find Nearest ";
+const DEFAULT_BUTTON_LABEL = "Location";
+
+export default class SimidMapCreative extends BaseSimidCreative {
+
+  constructor() {
+    super();
+  }
+
+  /** @override */
+  onStart(eventData){
+    debugger;
```

Comment:
I think you want to remove this line.

Reply:
yes sorry didn't realize I kept tjis one